src/app/feature/feature.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { FeatureRoutingModule } from './feature-routing.module';
+import { FeatureComponent } from './feature.component';
+
+
+@NgModule({
+  declarations: [
+    FeatureComponent
```

Comment:
What does this component implement?

Reply:
Literally nothing, I used Feature Module for lazy-loading of the CoursesComponent. And that Feature Module use in its structure template of FeatureComponent, I have deleted that component